deploy/production/ingress.yml

```diff
@@ -0,0 +1,53 @@
+apiVersion: networking.k8s.io/v1
+kind: Ingress
+metadata:
+  name: justice-gov-uk-production-ingress
+  namespace: justice-gov-uk-production
+  annotations:
+    external-dns.alpha.kubernetes.io/set-identifier: justice-gov-uk-production-ingress-justice-gov-uk-production-green
+    external-dns.alpha.kubernetes.io/aws-weight: "100"
+    nginx.ingress.kubernetes.io/auth-type: basic
+    nginx.ingress.kubernetes.io/auth-secret: basic-auth-secret
+    nginx.ingress.kubernetes.io/auth-realm: 'Production User | Authentication Required'
+    nginx.ingress.kubernetes.io/proxy-body-size: "200m"
```

Comment:
Do we know what this means??

Reply:
proxy-body-size is for allowing posing documents (zips) < 200 megs.

Ref: https://kubernetes.github.io/ingress-nginx/user-guide/nginx-configuration/configmap/#proxy-body-size

Looks like m (or M) is is ok to to use for megabytes https://nginx.org/en/docs/syntax.html